kasa/protocol.py

```diff
@@ -120,7 +140,8 @@ async def _query(self, request: str, retry_count: int, timeout: int) -> Dict:
         # ConnectionRefusedError) we do not want to keep trying since many
         # connection open/close operations in the same time frame can block
         # the event loop.
-        # This is especially import when there are multiple tplink devices being polled.
+        # This is especially important when multiple tplink devices are being
```

Comment:
Remove unrelated changes from this file.

Reply:
fixed